tests/virt_autotest/virt_utils.pm

```diff
@@ -143,5 +143,23 @@ sub update_guest_configurations_with_daily_build {
     # repl_guest_autoyast_addon_with_daily_build_module;
 }
 
+sub clean_up_red_disks {
+    my $wait_script           = "120";
+    my $get_disks_not_used    = "ls /dev/sd* | grep -v -e \"/dev/sd[a].*\" -e \"/dev/sd[b-z].*[[:digit:]].*\"";
+    my $disks_not_used        = script_output($get_disks_not_used, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my $get_disks_fs_overview = "lsblk -f";
+    my $disks_fs_overview     = script_output($get_disks_fs_overview, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my $get_fs_type_supported = "$get_disks_fs_overview | grep sda | awk \'{print \$2}\' | grep -v swap | tail -1";
+    my $fs_type_supported     = script_output($get_fs_type_supported, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my $make_fs_cmd           = "mkfs.$fs_type_supported";
+
+    if (($disks_not_used ne '') && ($fs_type_supported ne '')) {
```

Comment:
I guess we can simply use boolean context, as empty string is false in perl, so having:
`$disks_not_used  && $fs_type_supported`